packages/react-devtools-shared/src/devtools/views/Components/EditableValue.js

```diff
@@ -66,13 +76,23 @@ export default function EditableValue({
     }
   };
 
+  const applyChangesFromCheckBox = () => {
+    if (isValid) {
+      overrideValueFn(path, checkBoxRef.current.checked);
+    }
+  };
```

Comment:
I think we can just inline the override value part and get rid of this separate function :)